File: ios/Classes/plugin/domain/usecases/FeedLineUseCase.swift
import Foundation

public class FeedLineUseCase {
    private let repository: BluetoothRepository

    public init(repository: BluetoothRepository) {
        self.repository = repository
    }

    public func execute(callTimes: Int) -> Bool {
        return repository.printEmptyLine(callTimes: callTimes)
    }
}

File: android/src/main/kotlin/com/maktubcompany/easy_blue_printer/plugin/domain/usecase/FeedLineUseCase.kt
package com.maktubcompany.easy_blue_printer.plugin.domain.usecase

import com.maktubcompany.easy_blue_printer.plugin.domain.repository.BluetoothRepository

class FeedLineUseCase(private val repository: BluetoothRepository) {
    fun execute(callTimes: Int): Boolean {
        return repository.printEmptyLine(callTimes)
    }
}